### PR TITLE
adding fetchDeposits & fetchWithdrawals to cobinhood

### PR DESCRIPTION
**Pull request by lfern**
adding fetchDeposits & fetchWithdrawals to cobinhood (add parseTransactions to this class until added to base class.

### CODE REVIEW

File: js/cobinhood.js [hunk 1]
```diff
@@ -529,6 +529,82 @@ module.exports = class cobinhood extends Exchange {
...
+
+    parseTransactions (transactions, side = undefined, market = undefined, since = undefined, limit = undefined) {
+        let result = Object.values (transactions || []).map (transaction => this.parseTransaction (transaction, side));
+        result = this.sortBy (result, 'timestamp');
+        let symbol = (typeof market !== 'undefined') ? market['symbol'] : undefined;
+        return this.filterBySymbolSinceLimit (result, symbol, since, limit);
+    }
+
+    parseTransactionStatus (status) {
+        if (['tx_pending_two_factor_auth', 'tx_pending_email_auth', 'tx_pending_approval', 'tx_approved', 'tx_processing', 'tx_pending', 'tx_sent'].indexOf (status) !== -1) {
```
Comment (kroitor) [inline js/cobinhood.js]: we can't use indexOf in a portable way for arrays (we can use it for strings only), therefore these checks just have to be written out one by one in if-else cases, or in the same way as done in other places – by having a map of exchange-specific statuses to exchange-agnostic (unified) statuses.

Comment (kroitor) [reply js/cobinhood.js]: Example: https://github.com/ccxt/ccxt/blob/master/js/binance.js#L560

Comment (lfern) [reply js/cobinhood.js]: I see that parseTransactions method is not transpiled correctly to python. exchange.js parseTrades method is not really transpiled to python & php. So I suppose I must add parseTransactions method to both files (exchange.py, Exchange.php).

**Comment by lfern** [comment]
@kroitor Thanks! I think this is fixed. Please, let me know if I added ParseTransactions to Exchange.js, Exchange.php and exchange.py correctly.

**Comment by kroitor** [comment]
@lfern all looking ok at first glance, will probably rename a couple of fields for clarity upon merging, and from there we can scale it up to all exchanges in the same fashion then... Will merge very soon.

File: js/cobinhood.js [hunk 2]
```diff
@@ -529,6 +529,79 @@ module.exports = class cobinhood extends Exchange {
...
+        return {
+            'info': transaction,
+            'id': this.safeString (transaction, 'withdrawal_id'),
+            'txid': this.safeString (transaction, 'txhash'),
+            'timestamp': timestamp,
+            'datetime': datetime,
+            'currency': currency,
+            'status': this.parseTransactionStatus (transaction['status']),
+            'side': side, // direction of the transaction, ('deposit' | 'withdraw')
+            'price': undefined,
```
Comment (clarkmoody) [inline js/cobinhood.js]: Though it is in the sample data structure, I think that `price` should be removed from the common data format for transactions. I'm awaiting comment on #1896

Comment (clarkmoody) [reply js/cobinhood.js]: @lfern it looks like `price` should be removed. See discussion [here](https://github.com/ccxt/ccxt/issues/1896#issuecomment-391457166)

Comment (firepol) [reply js/cobinhood.js]: I submitted a PR fixing the price and other (tested) fixes but @lfern closed it (letting me a bit puzzled...)... so how should I proceed exactly to share my contribution? thx for letting me know...

Comment (kroitor) [reply js/cobinhood.js]: @lfern can you plz accept the edits submitted by @firepol ? ) 

@firepol if @lfern does not accept your edits, then you can either submit a standalone PR (I will merge the two), or I will cherry-pick your edits from the closed one.

Comment (firepol) [reply js/cobinhood.js]: @kroitor ok I just submitted a standalone PR that basically includes this PR + my fixes. I hope it works like that, cheers

Comment (kroitor) [reply js/cobinhood.js]: @firepol thx! I'm going to merge them this week, as promised.

**Pull request merged**